utils/misc/dbutils.go

```diff
@@ -10,6 +10,12 @@ import (
 	"github.com/rudderlabs/rudder-server/config"
 )
 
+type AdvisoryLock int
+
+const (
+	JobsDBAddDsAdvisoryLock = 11
```

Comment:
```suggestion
	JobsDBAddDsAdvisoryLock = 11
```
        JobsDBAddDsAdvisoryLock AdvisoryLock = 11